### PR TITLE
sbase: new recipe

### PR DESCRIPTION
**Pull request by extrowerk**
While the recipe builds fine, it fails at the install stage which i can't really understand. Please give me some hints.

Thank You!

### CODE REVIEW

**Comment by extrowerk** [comment]
Ah, it uses the freshly built mkdir and it doesn't want to create any directory if any parent folder is RO.

**Comment by Begasus** [comment]
When using `make sbase-box $jobArgs` in `BUILD()` and `make sbase-box-install PREFIX=$prefix MANPREFIX=$manDir` in `INSTALL()` (and some changes to the recipe and the Makefile) I got the package to build.
Using sbase-box will create one binary and links the other "binaries" to sbase-box.

File: sys-apps/sbase/sbase-0.0~git.recipe [hunk 1]
```diff
@@ -0,0 +1,251 @@
+SUMMARY="Portable core utilities"
+DESCRIPTION="A  collection of  unix  tools that  are inherently  portable \
```
Comment (diversys) [inline sys-apps/sbase/sbase-0.0~git.recipe:2]: Extra spaces

**Comment by extrowerk** [comment]
> ~When using `make sbase-box $jobArgs` in `BUILD()` and `make sbase-box-install PREFIX=$prefix MANPREFIX=$manDir` in `INSTALL()` (and some changes to the recipe and the Makefile) I got the package to build. Using sbase-box will create one binary and links the other "binaries" to sbase-box.~
> 
> **EDIT** strike that, it worked a minute ago, same issue now :(

Check what package got actvated in the chroot. Probably the previously built sbase causes the problem.

**Comment by Begasus** [comment]
> Check what package got actvated in the chroot. Probably the previously built sbase causes the problem.

Moving it out of the way did the trick. :)

**Comment by korli** [comment]
> Should we really add a recipe for this if it could potentially cause confusion in pkgman or HaikuPorter when looking for `cmd` provides?

it's a bad idea, at least not a HaikuPorts, this looks like a support nightmare.

**Comment by pulkomandy** [comment]
Not only a support nightmare, but wasn't the goal of Haiku to provide a unified system and avoid the modern GNU situation of "you have a dozen kernels, 10 different C libraries and 50 different implementations of the core utilities, please pick one of each to assemble your system"?

We should try to avoid that and pick one implementation for each tool. We can discuss which implementation is the best, but given this recipe can't even install itself, it seems not the best in the current state at least?

**Pull request closed without merging**